packages/typespec-vscode/snippets.json

```diff
@@ -0,0 +1,37 @@
+{
+  "Alias": {
+    "prefix": "alias",
+    "body": "alias ${1:AliasName} = ${0:Expression};",
+    "description": "Alias definition"
+  },
+  "Enum": {
+    "prefix": "enum",
+    "body": "enum ${1:EnumName} {\n  ${0:Value},\n}",
+    "description": "Enum definition"
+  },
+  "Interface": {
+    "prefix": "interface",
+    "body": "interface ${1:InterfaceName} {\n  ${2:member}(${3:parameters}): ${0:ReturnType};\n}",
+    "description": "Interface definition"
+  },
+  "Model": {
+    "prefix": "model",
+    "body": "model ${1:ModelName} {\n  ${2:propertyName}: ${0:propertyType};\n}",
+    "description": "Model definition"
+  },
+  "Namespace": {
+    "prefix": "namespace",
+    "body": "namespace ${1:NamespaceName} {\n  ${0:member}\n}",
+    "description": "Namespace definition"
+  },
+  "Operation": {
+    "prefix": ["op", "operation"],
+    "body": "operation ${1:OperationName}(${2:parameters}): ${0:ReturnType};",
```

Comment:
Maybe for operation, we can change the template to
```suggestion
    "body": "operation ${1:OperationName}(${0:parameters}): ${2:ReturnType};",
```
since the `parameters` could be multiple inputs

Reply:
On the other hand, the `ReturnType` could be a union.

Reply:
i would prefer to use the order people usually follow when coding. And we may follow the 'function' snippet of typescript with ReturnType added. 
![image](https://github.com/user-attachments/assets/e1a1dc72-2d0a-4449-a02c-3ba25bb57666)

Reply:
yeah I agree with Rodge I think it better to just stick to the order. As for the union I think its too opinionated.

Reply:
OK. I deleted `Alias` since it's too simple.

Reply:
changed to typescript function style